chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -43,8 +43,17 @@ def runningAsAdministrator(self):
         # 2003 and XP under normal mode.
         if 'win-2003' in self.hostname or 'win-xp' in self.hostname:
             return False
-        else:
-            return True
+
+        return True
```

Comment:
this is fine, but in the future I think that we should avoid doing edit wars, in which the code is reformatted without changing the logic :)